### ISSUE TITLE
There is an official documentation?

### ISSUE THREAD

**Issue by DevTony101** [aalmiray/bootstrapfx#12]
Although I know the basics of bootstrap for web development, is hard to know how to properly add classes to a jfx element. It would be a lot easier if there was a documentation page.

**Comment by aalmiray** [comment]
The README contains everything that there is to know about this Project. There's even an example setting a style on a `Button`. Is there anything else that's missing?

**Comment by DevTony101** [comment]
Yes, indeed, the README is very helpful and it shows everything the project currently supports, but it lacks code examples.

**Comment by aalmiray** [comment]
The current example is minimalistic in nature as its purpose is to show

  1. How the stylesheet may be applied. 
  2. How a style class may be applied to a graphical element. 

Everything in that snippet is regular JavaFX code BootstrapFX is just CSS at the moment (well except for a custom Pane class shown in the demo), there’s no additional API.

What more examples would be needed?

**Comment by DevTony101** [comment]
I was hoping there was at least one code example per graphical component (that is, one for buttons, split-buttons, label, alerts, etc) but then again, it is only a suggestion, the code example shown in the README is perfectly understandable.

**Comment by aalmiray** [comment]
Perhaps if every usage as you suggest were to be accompanied by a screenshot. However the snippets of code would quickly be repetitive.

**Comment by DevTony101** [comment]
I agree, but it would be very helpful.
